src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PublicSurfaceTests.cs

```diff
@@ -37,9 +37,9 @@ public void PublicSurfaceTest_Connectors()
               "Microsoft.PowerFx.Connectors.ConnectorParameters",
               "Microsoft.PowerFx.Connectors.ConnectorParameterWithSuggestions",
               "Microsoft.PowerFx.Connectors.ConnectorPermission",
+              "Microsoft.PowerFx.Connectors.ConnectorRelationships",
```

Comment:
>ConnectorRelationships [](http://example.com/codeflow?start=44&length=22)

can we keep these new types internal?

Reply:
Not this one for sure, see other comment